packages/theme/pages/Category.vue

```diff
@@ -517,12 +555,19 @@ export default defineComponent({
       facets.value = facetGetters.getGrouped(result.value, magentoConfig.facets.available);
       pagination.value = facetGetters.getPagination(result.value);
 
+      categoryTree.value = categoryGetters.getCategoryTree(
+        categories.value?.[0],
+        routeData.value?.entity_uid,
+        false,
+      );
+
       const tags = [{ prefix: CacheTagPrefix.View, value: 'category' }];
       // eslint-disable-next-line no-underscore-dangle
       const productTags = products.value.map((product) => ({ prefix: CacheTagPrefix.Product, value: product.uid }));
+      const categoriesTags = categoryTree?.value?.items?.map((category) => ({ prefix: CacheTagPrefix.Category, value: category.uid }));
```

Comment:
```suggestion
      const categoriesTags = categoryTree?.value?.items?.map((category) => ({ prefix: CacheTagPrefix.Category, value: category.uid })) ?? [];
```
can't spread a null, can be added to productTags in the line above aswell

Reply:
fixed